unime/src-tauri/gen/apple/ExportOptions.plist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <!-- Use "development" for local testing, use "ad-hoc" for TestFlight releases -->
+
+    <!-- <key>method</key>
+    <string>development</string>
+    <key>provisioningProfiles</key>
+	<dict>
+		<key>com.impierce.identity-wallet</key>
+		<string>UniMe (alpha) - Internal Test (Development)</string>
+	</dict> -->
+
+    <key>method</key>
+	<string>ad-hoc</string>
+	<key>provisioningProfiles</key>
+	<dict>
+		<key>com.impierce.identity-wallet</key>
+		<string>UniMe (alpha) - Internal Test (Distribution)</string>
```

Comment:
@daniel-mader , can we actually change <key>com.impierce.identity-wallet</key> to <key>com.impierce.unnime</key>  do you know?